biz.aQute.tester.junit-platform/src/aQute/tester/junit/platform/JUnitEclipseListener.java

```diff
@@ -41,6 +40,109 @@
 import org.opentest4j.MultipleFailuresError;
 
 public class JUnitEclipseListener implements TestExecutionListener, Closeable {
+
+	static final Class<? extends AssertionError>	JUNIT3_COMPARISON_FAILURE;
```

Comment:
I think these should be `Predicate<Object>`.

Reply:
I made them `Predicate<Throwable>` instead, but basically used your suggestion. Thanks for the idea, it does neaten up the code a bit.